Gemfile

```diff
@@ -3,7 +3,6 @@ source 'https://rubygems.org'
 plugin 'bundler-inject', '~> 1.1'
 require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundler-inject") rescue nil
 
-gem "catalog-api-client", :git => "https://github.com/mkanoor/catalog-api-client-ruby.git", :branch => "master"
```

Comment:
We also need to rework this area: https://github.com/RedHatInsights/catalog-api-minion/blob/master/lib/catalog/api/minion/base.rb#L74-L81 since it uses the client.

The thought was that we should manually parse the host from the deployment config environment variable.